examples/rosettacode/guess_the_number.nit

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: guess the number
+# SEE: <http://rosettacode.org/wiki/Guess_the_number>
+#
+
+var random = 10.rand +1
+var str = 0
+while str != random do
+	print "enter a guess :"
+	str = gets.to_i
```

Comment:
you should protect the end of file to avoid infinite loops.

~~~nit
if stdin.eof then break
~~~